src/layer/AbstractSentinelHubV3Layer.ts

```diff
@@ -397,4 +396,9 @@ export class AbstractSentinelHubV3Layer extends AbstractLayer {
     const res = await axios.post(url, evalscript, requestConfig);
     return res.data;
   }
+
+  protected getConvertScriptBaseUrl(): string {
```

Comment:
Just for consistency, can we maybe name this `getConvertEvalscriptBaseUrl`? Fftr.

Reply:
Was renamed